docker-compose.yml

```diff
@@ -10,7 +10,7 @@ services:
         environment:
             - "BTC_RPCUSER=dappnode"
             - "BTC_RPCPASSWORD=dappnode"
-            - "BTC_TXINDEX=1"
-            - "BTC_PRUNE=0"
+            - "BTC_TXINDEX=0"
```

Comment:
Why are you enabling pruning by default? Should be zero, and optional

Reply:
Fixed, I thought you wanted a separate package actually